src/manifest/dbtProject.ts

```diff
@@ -639,6 +640,12 @@ select * from renamed
   }
 
   private findTargetPath(projectConfig: any): string {
+    const targetPathOverride = workspace
+      .getConfiguration("dbt")
+      .get<string>("targetPathOverride");
+    if (targetPathOverride !== undefined) {
+      return substituteSettingsVariables(targetPathOverride);
+    }
```

Comment:
Could you also add support for getting the target path from DBT_TARGET_PATH environment variable?

Reply:
Sure I can add that in.

Reply:
I pattern matched what the `DBT_PROFILE_DIR` logic was doing but I was having a hard time getting vscode to recognize the env variable with `terminal.integrated.env.osx`. I was able to test that the override setting still worked and that if nothing was set it would default to `target`.